src/metabase/automagic_dashboards/core.clj

```diff
@@ -841,7 +855,7 @@
                                      (encode-base64-json
                                       {:database (:database root)
                                        :type     :query
-                                       :query    {:source_table (->> root
+                                       :query    {:source-table (->> root
```

Comment:
How does this PR change where/when/how `snake_case` gets converted to `kabob-case`? Does the FE now use `kabob-case`?

Reply:
Yes, `Card.dataset_query` and other query stuff gets normalized when it comes out of the database. The FE uses lisp-case now since it's guaranteed to be in that normalized format

Reply:
Only queries are affected, everything else is the same